bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -89,12 +92,13 @@ public static String executeUrl(String httpMethod, String url, int timeout) thro
      * Furthermore the <code>http.proxyXXX</code> System variables are read and
      * set into the {@link HttpClient}.
      *
-     * @param httpMethod the HTTP method to use
-     * @param url the url to execute
-     * @param content the content to be send to the given <code>url</code> or <code>null</code> if no content should be
-     *            send.
+     * @param httpMethod  the HTTP method to use
+     * @param url         the url to execute
+     * @param content     the content to be send to the given <code>url</code> or <code>null</code> if no content should
+     *                        be
+     *                        send.
```

Comment:
and while you are at it, please also fix the old typo send->sent.

Reply:
Done